### PR TITLE
[QE-4033] Add module creation and build tests

### PR DESCRIPTION
**Pull request by longton95**
https://techweb.axway.com/jira/browse/QE-4033

### CODE REVIEW

File: Helpers/Appc_Helper.js [hunk 1]
```diff
@@ -97,6 +96,20 @@ class Appc_Helper {
...
 	}
+	static checkModule(moduleName, platform, packageName) {
+
+		platform = platform.toLowerCase();
+
+		let
+			rootPath = path.join(global.workspace, moduleName),
+			zipPath = (platform === 'ios') ? `${rootPath}/${platform}/${packageName}-iphone-1.0.zip` : `${rootPath}/${platform}/dist/${packageName}-${platform}-1.0.zip`;
+
+		if (fs.existsSync(zipPath)) {
```
Comment (ewanharris) [inline Helpers/Appc_Helper.js]: Could this just be `fs.existsSync(zipPath)`?

File: Tests/Mac-iOS/4.Generate_Module.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const
+	assert = require('assert'),
+	Appc = require('../../Helpers/Appc_Helper.js'),
+	MochaFilter = require('mocha-filter')(global.filters);
+
+describe.only('Generate Module', () => {
```
Comment (SamJHaig) [inline Tests/Mac-iOS/4.Generate_Module.test.js]: Remove `.only`

File: Tests/Mac-iOS/5.Build_Module.test.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const
+	assert = require('assert'),
+	Appc = require('../../Helpers/Appc_Helper.js'),
+	MochaFilter = require('mocha-filter')(global.filters);
+
+describe.only('Build Module', () => {
```
Comment (SamJHaig) [inline Tests/Mac-iOS/5.Build_Module.test.js]: Remove `.only`

File: Helpers/Appc_Helper.js [hunk 4]
```diff
@@ -125,10 +129,40 @@ class Appc_Helper {
...
-	static checkGenerated() {
+	static checkModule() {
+
+		let platform = global.platformOS.toLowerCase();
+
+		const
+			rootPath = genRootPath(mod.name),
+			zipPath = (platform === 'ios') ? path.join(rootPath, platform, 'dist', `${mod.packageName}-iphone-1.0.0.zip`) : path.join(rootPath, platform, 'dist', `${mod.packageName}-${platform}-1.0.0.zip`);
+
+		console.log(zipPath);
```
Comment (SamJHaig) [inline Helpers/Appc_Helper.js]: Remove `console.log()`

**Pull request merged**